OrbitClientData/FunctionInfoSetTest.cpp

```diff
@@ -0,0 +1,274 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include "OrbitClientData/FunctionInfoSet.h"
+#include "capture_data.pb.h"
+
+using orbit_client_protos::FunctionInfo;
+
+TEST(FunctionInfoSet, EqualFunctions) {
+  FunctionInfo left;
+  left.set_name("foo");
+  left.set_pretty_name("void foo()");
+  left.set_loaded_module_path("/path/to/module");
+  left.set_module_base_address(42);
+  left.set_address(12);
+  left.set_load_bias(4);
+  left.set_size(16);
+  left.set_file("file.cpp");
+  left.set_line(13);
+
+  FunctionInfo right;
+  right.CopyFrom(left);
```

Comment:
maybe just copy and paste in this particular case lines 13-22 for right?

Reply:
sure. Done